### ISSUE TITLE
New GiD Events - Delete deprecations

### ISSUE THREAD

**Issue by jginternational** [KratosMultiphysics/GiDInterface#444]
GiD has some new events, and the old ones are deprecated.

The old ones still work but not properly: Toolbars that are not removed when changing to a problemtype, windows that does not appear...

Since GiD 14.1.1d (few days remaining), GiD will start showing this message to all the kratos problemtype users:
```
Implemented 12 events deprecated:
AfterMeshGeneration -> GiD_Event_AfterMeshGeneration
AfterRenameGroup -> GiD_Event_AfterRenameGroup
AfterTransformProblemType -> GiD_Event_AfterTransformProblemType
AfterWriteCalcFileGIDProject -> GiD_Event_AfterWriteCalculationFile
BeforeRunCalculation -> GiD_Event_BeforeRunCalculation
BeforeTransformProblemType -> GiD_Event_BeforeTransformProblemType
ChangedLanguage -> GiD_Event_ChangedLanguage
EndGIDPostProcess -> GiD_Event_EndGIDPostProcess
EndGIDProject -> GiD_Event_EndProblemtype
InitGIDPostProcess -> GiD_Event_InitGIDPostProcess
InitGIDProject -> GiD_Event_InitProblemtype
SaveGIDProject -> GiD_Event_SaveModelSPD
```
It's possible that, when changing to these new events, we'll need to upgrade the MinimumGiDVersion to 14, so any user with GiD 13 will need to upgrade it's license.

@KratosMultiphysics/technical-committee

**Comment by josep-m-carbonell** [comment]
Does it afect to the current Kratos Release? Do you think it is better to not allow downloading the release to a version of GiD that presents incompatibilities?

**Comment by RiccardoRossi** [comment]
@jginternational do we really need to give up compatibility with Gid13? ... upgrading the licenses is quite a big setback ...

**Comment by pooyan-dadvand** [comment]
In my opinion it is too early to drop the support for GiD 13.....

I would maintain it for more time.

**Comment by jginternational** [comment]
@pooyan-dadvand 
Users will still have available the released Kratos 6.0 in the GiD Internet retrieve for GiD 13. 

Only developers will be affected by this change

**Comment by josep-m-carbonell** [comment]
I do not understand this:
'The old ones still work but not properly: Toolbars that are not removed when changing to a problemtype, windows that does not appear...'
I guess that with GiD 14.1.1d the relesase Kratos 6.0 will work in the same way it was for GiD 13. Isn't it?
